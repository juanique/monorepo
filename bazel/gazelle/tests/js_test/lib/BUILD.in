load("//bazel/js:js.bzl", "js_library")

# This file intentionally empty - Gazelle should generate the appropriate rule
