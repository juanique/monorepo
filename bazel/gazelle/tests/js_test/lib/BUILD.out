load("//bazel/js:defs.bzl", "js_library")

js_library(
    name = "lib",
    srcs = ["utils.js"],
)
